custom/src/main/java/org/kiwix/kiwixmobile/custom/main/CustomReaderFragment.kt

```diff
@@ -69,23 +66,17 @@ class CustomReaderFragment : CoreReaderFragment() {
   @Inject lateinit var sharedPreferenceUtil: SharedPreferenceUtil
   @Inject lateinit var dialogShower: DialogShower
 
-  override fun onCreateView(
-    inflater: LayoutInflater,
-    container: ViewGroup?,
-    savedInstanceState: Bundle?
-  ): View? {
-    val view = super.onCreateView(inflater, container, savedInstanceState)
-
+  override fun onActivityCreated(savedInstanceState: Bundle?) {
```

Comment:
Did you test customApp after this change?

Reply:
Yes, it works. It didn't when the setup was done in onCreateView because CoreReaderFragment inits in onActivityCreated.

Reply:
excellent